test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TestHtmlEncoder.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Framework.WebEncoders;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    public class TestHtmlEncoder : IHtmlEncoder
```

Comment:
`NullHtmlEncoder` falls same format as the logging bits that just went into testing.